install.sh

```diff
@@ -65,6 +65,20 @@ function backup_new() {
   echo -e "\033[0;32mCopied the template $CONFIG_FILE into ~/$CONFIG_FILE, edit this file to customize bash-it\033[0m"
 }
 
+# Append a target file so that it sources .bashrc
+function setup_interactive_dotfile() {
+tee -a "${1:?No Target Specified}" <<'EOF' >/dev/null
```

Comment:
I don't think there is a need to call out to `tee` when this check can be done native in bash.

Reply:
This comment is puzzling.  Can you please explain your concern more thoroughly?

Reply:
Sorry two thoughts I conflated when making the quick comment.

1) the test for the parameter should be it's own line

2) `tee`'s purpose is to split output to a file and stdout, but you are redirecting stdout to /dev/null. Just seems like a superfluous use. `cat` would be a better fit, but the best would be the `printf` built-in.